### ISSUE TITLE
💡 Vocabulary Structure Proposal

### ISSUE THREAD

**Issue by markdoerr** [nfdi4cat/voc4cat#33]
### Description

For the overall structure of the repository I would suggest the following:
General approach: from generic / common terms to specific (collecting most  generic terms in a kind of "top-level vocabulary") - each term should be defined, if possible, only once, and as broad as possible:

1. general chemistry / physics ( molecules, reactions, physical measurements, light, absorbance, ...)
2. catalysis (terms that are in common to all areas of catalysis (catalyst, .... )
3. one vocabulary for each catalytic field: heterogen. cat., homogen., biocatalysis, photocatalysis, electrocatalysis
4. process and process modelling vocabulary (reactors,  heat-exchange, material exchange, ...)

(I will create the corresponding sections in the config file)

**Comment by dalito** [comment]
We should discuss with @nmoust . - I proposed to him some guidelines how to model (in private). For example we should use broader/narrower for *is_a* relation put not for *part_of*. *part_of* is collection territory.

**Comment by dalito** [comment]
@markdoerr - #48 is the result of what @nmoust and I had worked on internally. We propose few top level concepts and give some reasons for the proposed structure.

**Comment by dalito** [comment]
Closing this in favor of #48. If required we can later create specific new issues later (after implementing the hierarchy changes of related to #48).
